conf/BUILD

```diff
@@ -0,0 +1,60 @@
+files(
+    name="st2client_sample_config",
+    sources=["st2rc.sample.ini"],
+)
+
+file(
+    name="st2.conf.sample",
+    source="st2.conf.sample",
+)
```

Comment:
A future PR will add `pants-plugins/sample_conf` which will wire this up so that it gets regenerated whenever someone runs the `fmt` goal with something like: `./pants fmt conf/st2.conf.sample`. That will take advantage of the fine-grained caching so that this always gets updated when the files used to generate it change.